app/controllers/application_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ApplicationController < ActionController::Base
+  # Prevent CSRF attacks by raising an exception.
+  # For APIs, you may want to use :null_session instead.
+  protect_from_forgery with: :exception
+
+  def index
+    redirect_to ideas_path
+  end
+
+  def current_user
+    @_current_user ||= User.find(session[:user_id]) if session[:user_id]
+  end
+
+  def find_idea(id)
```

Comment:
if you are only going to call this from the IdeasController, then you should just add this as a private method within the IdeasController